contrib/docker-entrypoint.sh

```diff
@@ -23,4 +23,8 @@ if [ "$1" = 'collector' ]; then
   eval $CMD_PREFIX /home/pganalyze/collector --statefile=/state/pganalyze-collector.state --no-log-timestamps
```

Comment:
Note that this may be [bash specific syntax](https://stackoverflow.com/questions/3811345/how-to-pass-all-arguments-passed-to-my-bash-script-to-a-function-of-mine) and I haven't tested this -- notably the entrypoint right now specifies `/bin/sh` at the top, not `/bin/bash`.

Reply:
Uhm, I'm not sure about this.
This does give more flexibility, though we still limit the statefile path and `no-log-timestamps` option, also this could lead the mix usage of args and env var for various configs (which won't work well I think). Given that we have `eval $CMD_PREFIX "$@"` at the end already for folks to do whatever they want to, I think the preset commands are good to be a safe guard with limited usage side, rather than giving more flexibility.

Some alternate to make verbose more generic (to support test with -v too, which is a good point), how about accepting `verbose` as a second argument? Like the following, and update the each "preset" commands to have this suffix:

```sh
CMD_SUFFIX=""
if [ "$2" = 'verbose' ]; then
  CMD_SUFFIX="--verbose"
fi
```

Also just FYI, `{@:2}` is indeed a bash specific syntax and I think alpine image doesn't have bash (quick tested and it didn't work).

Reply:
In our collector setting we tried initially to use `collector -v` but got trapped by this [if statement](https://github.com/pganalyze/collector/blob/61a4ae8f22260f91907db2f35a354f4c1a0f279d/contrib/docker-entrypoint.sh#L22-L24) that replaced the all command. I think it could be a good idea if other args are kept.